Spigot-Server-Patches/0616-add-get-set-biome-for-namespaced-keys.patch

```diff
@@ -0,0 +1,96 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Thonk <30448663+ExcessiveAmountsOfZombies@users.noreply.github.com>
+Date: Sun, 13 Dec 2020 23:12:42 -0600
+Subject: [PATCH] add get/set biome for namespaced keys
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/CraftWorld.java b/src/main/java/org/bukkit/craftbukkit/CraftWorld.java
+index b71316cce3bdbf3485be456f0260c6b3463cff8e..e57703abb8981e887d85e291eaac2b9cdbd6036e 100644
+--- a/src/main/java/org/bukkit/craftbukkit/CraftWorld.java
++++ b/src/main/java/org/bukkit/craftbukkit/CraftWorld.java
+@@ -103,6 +103,7 @@ import org.bukkit.Effect;
+ import org.bukkit.FluidCollisionMode;
+ import org.bukkit.GameRule;
+ import org.bukkit.Location;
++import org.bukkit.NamespacedKey;
+ import org.bukkit.Particle;
+ import org.bukkit.Raid;
+ import org.bukkit.Sound;
+@@ -1087,6 +1088,28 @@ public class CraftWorld implements World {
+         return CraftBlock.biomeBaseToBiome(getHandle().r().b(IRegistry.ay), this.world.getBiome(x >> 2, y >> 2, z >> 2));
+     }
+ 
++    // Paper start
++    @Override
++    public NamespacedKey getBiomeKey(int x, int y, int z) {
++        return CraftBlock.biomeBaseToKey(getHandle().r().b(IRegistry.ay), this.world.getBiome(x >> 2, y >> 2, z >> 2));
```

Comment:
obfhelpers please

Reply:
Ok, I think I added the helpers properly, needed to bring in two new classes and fix the errors before they'd let me compile it back together.